buildSrc/src/main/kotlin/otel.japicmp-conventions.gradle.kts

```diff
@@ -54,6 +54,16 @@ class AllowNewAbstractMethodOnAutovalueClasses : AbstractRecordingSeenMembers()
 class SourceIncompatibleRule : AbstractRecordingSeenMembers() {
   override fun maybeAddViolation(member: JApiCompatibility): Violation? {
     if (!member.isSourceCompatible()) {
+
+      // TODO: remove after 1.36.0 is released, see https://github.com/open-telemetry/opentelemetry-java/pull/6248
+      if (member.compatibilityChanges.isEmpty() &&
+        member is JApiClass &&
+        member.newClass.get().name.equals("io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder")
```

Comment:
you might be able to narrow this down a bit, see https://github.com/open-telemetry/opentelemetry-java/commit/f6deb4c1b7b9ed250f3f7cda992a012b6e738af2

Reply:
thanks for this real-life intro to japicmp 😉 I think I narrowed it down fully now.